doc/Administration-guide/System-installation/Single-node-installation/index.md

```diff
@@ -622,40 +955,56 @@ Add the MongoDB repository to the /etc/apt/sources.list.d/mongodb.list.
 To install Kaa you will need to [download](http://www.kaaproject.org/download-kaa/) pre-built packages or [build](../Planning-your-deployment#building-kaa-from-source) them from the [source code](https://github.com/kaaproject/kaa). We will use pre-built packages in this guide.
 
 <ul class="nav nav-tabs">
-  <li class="active"><a data-toggle="tab" href="#Ubuntu1">Ubuntu 14.04</a></li>
-  <li><a data-toggle="tab" href="#CentOS1">CentOS 6.7</a></li>
+  <li class="active"><a data-toggle="tab" href="#Ubuntu16_">Ubuntu 16.04</a></li>
+  <li><a data-toggle="tab" href="#Ubuntu14_">Ubuntu 14.04</a></li>
+  <li><a data-toggle="tab" href="#CentOS_">CentOS 6.7</a></li>
 </ul>
 
-<div class="tab-content">
-<div id="Ubuntu1" class="tab-pane fade in active" markdown="1" >
+<div class="tab-content"><div id="Ubuntu16_" class="tab-pane fade in active" markdown="1" >
+
+1. Download the latest Debian package from the [Kaa download page](http://www.kaaproject.org/download-kaa/).
+2. Unpack the downloaded tarball by executing the following command.
+
+   ```bash
+   $ tar -xvf kaa-deb-*.tar.gz
+   ```
+
+
+3. Install the Node service by executing the following command.
+
+   ```bash
+   $ sudo dpkg -i kaa-node.deb
+   ```
+
+</div><div id="Ubuntu14_" class="tab-pane fade" markdown="1" >
 
 1. Download the latest Debian package from the [Kaa download page](http://www.kaaproject.org/download-kaa/).
 2. Unpack the downloaded tarball by executing the following command.
 
    ```bash
-    $ tar -xvf kaa-deb-*.tar.gz
+   $ tar -xvf kaa-deb-*.tar.gz
    ```
 
 
 3. Install the Node service by executing the following command.
 
    ```bash
-    $ sudo dpkg -i kaa-node.deb
+   $ sudo dpkg -i kaa-node.deb
    ```
 
-</div><div id="CentOS1" class="tab-pane fade" markdown="1" >
+</div><div id="CentOS_" class="tab-pane fade" markdown="1" >
 
 1. Download the latest RPM package from the [Kaa download page](http://www.kaaproject.org/download-kaa/).
 2. Unpack the downloaded tarball by executing the following command.
 
    ```bash
-    $ tar -xvf kaa-rpm-*.tar.gz
+   $ tar -xvf kaa-rpm-*.tar.gz
    ```
 
 3. Install the Node service by executing the following command.
 
    ```bash
-    $ sudo rpm -i kaa-node.rpm
+   $ sudo rpm -i kaa-node.rpm
    ```
 
 </div></div>
```

Comment:
'SQL database configuration' section should be updated in order to give clear instructions about specific SQL DB configuration. Use separate tabs for MariaDB and PostgreSQL with detailed information about configuration files that should be updated and specific jdbc properties.